<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCar</class>
 <widget class="QWidget" name="FormCar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButton_LeftTurn">
       <property name="text">
        <string>Turn left</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Hazards</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_RightTurn">
       <property name="text">
        <string>Turn right</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Doors</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Left Front</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Boot</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Right Front</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Hood</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>Left Rear</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>Right Rear</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Speed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_Speed">
           <property name="maximum">
            <number>240</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_Speed">
         <property name="maximum">
          <number>240</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>RPM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_RPM">
           <property name="maximum">
            <number>8000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_RPM">
         <property name="maximum">
          <number>8000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Fuel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_Fuel"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_Fuel">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>TEMP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_Temp">
           <property name="minimum">
            <number>-60</number>
           </property>
           <property name="maximum">
            <number>195</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_EngTemp">
         <property name="minimum">
          <number>-60</number>
         </property>
         <property name="maximum">
          <number>195</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_Fuel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Fuel</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_RPM</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_RPM</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_Fuel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_Fuel</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_Speed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_Speed</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_Temp</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_EngTemp</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_EngTemp</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Temp</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Speed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Speed</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_RPM</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_RPM</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
